components/popover_list_members/popover_list_members.jsx

```diff
@@ -124,9 +125,9 @@ export default class PopoverListMembers extends React.Component {
                             key={'popover-member-' + i}
                         >
                             <ProfilePicture
+                                className='user__picture'
                                 src={Client4.getProfilePictureUrl(m.id, m.last_picture_update)}
-                                width='40'
-                                height='40'
+                                status={getStatusForUserId(state, m.id)}
```

Comment:
My bad for not clarifying. I wasn't sure if you'd worked with Redux before, so I can give you a quick rundown of how it should work.

Basically, we want the container object ([`components/popover_list_members/index.js`](https://github.com/mattermost/mattermost-webapp/blob/master/components/popover_list_members/index.js)) to be the only thing that talks directly to the store. It will then pass anything that this component needs to render as part of the props. This makes eventually testing this function really easy since you can test it directly modifying the stores. If you look at the `mapStateToProps` function in that file, it's used to read from the redux store and set the props for this component. So what you'll want to do is:
1. Add status to the object returned by `mapStateToProps` so that it gets passed in as `this.props.status`
2. Add the prop type as you would with any other prop
3. Use `this.props.status` here

Reply:
Thanks for taking the time to explain that, very helpful, hopefully it looks alright now :)